.ci/daily-config.yaml

```diff
@@ -4,25 +4,99 @@ dependencies: ./project-dependencies.yaml
 
 pre: |
   export MAVEN_COMMAND=mvn -B -e -U clean deploy -Dfull -Drelease -DaltDeploymentRepository=local::default::file://${{ env.WORKSPACE }}/deploy-dir -Dmaven.test.redirectTestOutputToFile=true -Dmaven.test.failure.ignore=true -Dgwt.memory.settings="-Xmx10g" -Prun-code-coverage -Dcontainer.profile=wildfly -Dcontainer=wildfly -Dintegration-tests=true -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
-  echo "MAVEN_COMMAND=" ${{ env.MAVEN_COMMAND }}
-  echo "WORKSPACE=" ${{ env.WORKSPACE }}
-  java -version
```

Comment:
I would keep the following line:
```
  java -version
```
it could be useful when debugging and even the `mvn` version would be useful IMO (`mvn --version`)

Reply:
yes - first has to be done the version update and then the mvn build/deploy - this is correct - will re-add java-version